### DISCUSSION TITLE
Proper diagram is not visible while using Export to vsdx using react tsx

### DISCUSSION THREAD

**Discussion by milenCroy** [clientIO/joint#2644] [Q&A]
### Introduction

By using the example given in node ts in JointJS, I had tried to replicate my requirements in react tsx. 
But while doing so, Only connecting lines of diagram is visible. Boxes and tables are not visible. 

Can you provide an example to use "Export to vsdx" in reaxt tsx So it will be much helpful for my requirment.
  
I opened the file in Visio. Screenshot is attached below.
![thumbnail_image](https://github.com/clientIO/joint/assets/169026692/cf51fd02-2092-4de3-aac4-fc06e3c2c4f0)

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment]
We do not have an example of exporting to Visio in ReactTS. In addition, the framework used (React, Angular, ...) does not affect the functioning of the Visio export plugin.

If the diagram is not exported as expected you can report it as a bug along with all the information for us to be able to reproduce  it.

**Comment by milenCroy** [comment]
Can you share GitHub with Visio integrated in react.

**Comment by kumilingus** [comment]
The problem was that the customer tried to map the JointJS shape to a non-existent Visio Master shape.
